src/rpc.ts

```
@@ -0,0 +1,94 @@
+    success = 0;
+    /** Number of unsuccessful requests */
+    failure = 0;
+    /** Last request timestamp in milliseconds unix format */
+    lastRequestTimestamp = 0;
+    /** List of times between 2 consecutive requests in milliseconds */
+    requestIntervals: number[] = [];
+    /** A utility cache, that can hold any data betwen separate requests */
+    cache: Record<string, any> = {};
+
+    /** Creates a new instance */
+    constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)